static/js/components/ProgressWidget.js

```diff
@@ -61,6 +62,19 @@ export const circularProgressWidget = (
   )
 }
 
+export const gradeRecordsLink = (url: string): React$Element<*> => {
+  return (
+    <a
+      className="mm-minor-action"
+      href={url}
+      target="_blank"
+      rel="noopener noreferrer"
+    >
+      View Grade Records
```

Comment:
Let's change this to "View Program Record" to be consistent with edX. 

I'm still thinking about the paragraph of text that edX's program dashboard has. I don't really want to add it, but maybe we will have to.